pkg/mountutil/mountutil_windows.go

```diff
@@ -14,10 +16,19 @@
    limitations under the License.
 */
 
+/*
+   Portions from https://github.com/moby/moby/blob/master/volume/mounts/windows_parser.go#L53
```

Comment:
Oh I think this is the attribution, can you use a tagged/branch version instead of the `master` branch?